resources/views/profile.blade.php

```diff
@@ -44,21 +47,82 @@
-                                <td>Active expenses: </td>
-                                <td></td>
-                            </tr>
-                            <tr class="active">
-                                <td>Total expenses: </td>
-                                <td></td>
-                            </tr>
-                            <tr class="active">
-                                <td>Total friends: </td>
-                                <td></td>
+                                <td>Biography: </td>
+                                <td>{{ Auth::user()->bio }}</td>
                             </tr>
                         </table>
+                        <button class="btn btn-success openEditProfileModal" data-toggle="modal" data-target="#editProfileModal">Edit Profile</button>
                     </div>
                 </div> <!-- row -->
             </div> <!-- container-fluid -->
         </div> <!-- main-panel -->
     </div> <!-- wrapper -->
-@endsection
+
+    <div class="modal fade openEditProfile" id="editProfileModal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span></button>
+                    <h4 class="modal-title" id="editProfileLabel">Edit Profile</h4>
+                </div>
+                <form id="editProfileForm" action="/profile/{{ Auth::user()->username }}/edit" method="POST" class="form-horizontal">
+                    {{csrf_field()}}
+                    <div class="modal-body">
+                        <div class="row">
+                            <div class="col-md-6">
+                                {!! Form::label('username', 'Username:') !!}
+                                <input type="text" name="username" id="modal_username" class="form-control" value="{{ Auth::user()->username }}" readonly="true">
+
+                                {!! Form::label('firstname', 'First name:') !!}
+                                <input type="text" name="firstname" id="modal_firstname" class="form-control" value="{{ Auth::user()->firstname }}" required="true">
+
+                                {!! Form::label('lastname', 'Last name:') !!}
+                                <input type="text" name="lastname" id="modal_lastname" class="form-control" value="{{ Auth::user()->lastname }}" required="true">
+
+                                {!! Form::label('email', 'Email:') !!}
+                                <input type="email" name="email" id="modal_email" class="form-control" value="{{ Auth::user()->email }}" required="true">
+                            </div>
+                            <div class="col-md-6 text-center">
+                                <br>
+                                <?php
```

Comment:
Really minor. But could we use @php - @endphp to enclose php code blocks just to keep the convention and